src/wallet/models.py

```diff
@@ -92,4 +156,19 @@ def countable(self):
         For example, rejected transactions do not contribute to the wallet
         balance.
         """
-        return self.trx_type != enums.TrxType.PENDING
+        currency = self.trx.wallet.currency
+        if self.trx.amount < Money(0, currency or settings.DEFAULT_CURRENCY):
+            # Outgoing
+            ACCEPTED = (enums.TrxStatus.PENDING, enums.TrxStatus.CANCELLATION)
```

Comment:
It could probably be easier to just write
```python
return self.status in (enums.TrxStatus.PENDING, 
                       enums.TrxStatus.CANCELLATION)
```
It makes it easier to follow what's going on.